web_server/posts/templates/posts/post.html

```diff
@@ -0,0 +1,110 @@
+<!-- reference: https://www.w3schools.com/bootstrap/tryit.asp?filename=trybs_temp_blog&stacked=h -->
+{% extends "navigation.html" %}
+{% block navigation_content %}
+{% load static %}
+
+
+
+<div class="container-fluid">
+     <div class="row">
+        <div class="col-sm-2">
+
+        </div>
+         <div class="col-sm-8">
+             <br>
+             <br>
+
+             <div class="form-inline">
+                 <div class="d-flex w-100 justify-content-between">
+                     <h2>{{post.title}}</h2>
```

Comment:
Post description isn't anywhere on the page, I think this is something that should be displayed.

Reply:
The description has been added - the look might need to be changed later.